waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -1672,4 +1674,15 @@ public CompactionResponse compact2(CompactionRequest rqst) throws TException {
     return mapping.getClient().compact2(mapping.transformInboundCompactionRequest(rqst));
   }
 
+  // Hive 2.3.7 methods
+
+  @Override
+  @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
+  public PartitionValuesResponse get_partition_values(PartitionValuesRequest req) throws MetaException, NoSuchObjectException, TException {
+    DatabaseMapping mapping = databaseMappingService.databaseMapping(req.getDbName());
+    return mapping
```

Comment:
I think this result needs to go through the missing `transformOutboundPartitionValuesRequest` method I mentioned earlier.

Reply:
OK, so since the response doesn't contain the database name this isn't needed.